php/libraries/Module.class.inc

```diff
@@ -62,6 +62,51 @@ abstract class Module extends \LORIS\Router\PrefixRouter
         return $modules;
     }
 
+    /**
+     * Checks whether one or more modules are active in the given database.
+     *
+     * @param \Database $db      an open connection to a database containing a
+     *                           'modules' table
+     * @param array     $modules the names of the modules to check
+     *
+     * @return array of module name => boolean (active or not)
+     */
+    static public function checkActiveModules(\Database $db, array $modules): array
```

Comment:
This looks like it's basically duplicating the functionality of LorisInstance->hasModule from the master branch. Since it's not a critical bug fix, I think it would be better for this to go to master and use that functionality rather than duplicate the functionality in a static function that would be deprecated right from the start..

Reply:
This function should be more efficient than `LorisInstance->hasModule` because it doesn't check all information. This is exactly the purpose of adding a new function. Otherwise I could have used the existing `getActiveModules` which is considered by me as not efficient enough to be useful for this PR's purpose.

Reply:
What were the results when you benchmarked hasModule and getActiveModules compared to this function?

Reply:
I don't have the number, `hasModule` and `getActiveModules` will loop around the whole modules, and check whether it exists or not, `checkActiveModules` only checks one or a few modules active or not in the database, this should not be in the same performance level.

Reply:
I just manually did a comparison, out of two executions, my method with `checkActiveModules` will takes 0.000150442s, compared with 0.021488905s of `hasModule` and `getActiveModules`. @driusan

Reply:
Assuming n=1 gives accurate statistics it's not worth the maintenance overhead of having 2 methods which do essentially the same thing in 2 different places in order to save 0.02s per page load of the imaging browser.